hbase-server/src/test/java/org/apache/hadoop/hbase/master/normalizer/TestRegionNormalizerWorkQueue.java

```diff
@@ -203,16 +202,17 @@ public void testTake() throws Exception {
       Thread.sleep(10);
       queue.put(i);
     }
-
+    // get some sleep to let consumer consume the last one
+    Thread.sleep(10);
```

Comment:
Is it possible to use HBTU.waitFor to make the test more reliable?

Reply:
Okay, I've revised it